### DISCUSSION TITLE
Need help for adding Hermes

### DISCUSSION THREAD

**Discussion by sakthivel-servicemax** [microsoft/react-native-windows#8980] [Q&A] [answered]
Hi, My current react native windows version is "react-native-windows": "^0.63.0-0",
I am upgrading this to "react-native-windows": "0.65.4",

I want to enable hermes for my project. My start up project is in C# and, it does not have packages.config and ExperimentalFeatures.props. 

I need help on adding hermes to my project. and will it make performance improvement on page loading?

Thanks,
Sakthi.

**Comment by chrisglein** [comment, marked as answer]
The `ExperimentalFeatures.props` was added for C# apps in 0.65. So a new app would get this when instantiating the template from the CLI. But since you upgraded a new app you're likely missing some changes. There is the upgrade helper tool, but it currently doesn't yet support C# apps, so that wouldn't have helped you. The main mitigation here is to generate a new blank app (0.65 or newer) and then copy over all those changes to your app (or the other way around). It's a bit manual, but that would unblock you.
